src/components/launchpad/upgrade-membership.tsx

```
@@ -0,0 +1,74 @@
+    authenticatedUser: any;
+}
+
+const UpgradeMembership: React.FC<UpgradeMembershipProps> = ({ authenticatedUser }) => {
+
+    const handleCallCheckoutSession = async () => {
+        try {
+            const result = await createCheckoutSession({
+                email: authenticatedUser.user.email
+            })
+
+            if (result && result.url) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)